### DISCUSSION TITLE
How to attach directories to android.apk and load them（android.apkにディレクトリを添付し、それらを読み込む方法）

### DISCUSSION THREAD

**Discussion by HirokatsuIshikawa** [airsdk/Adobe-Runtime-Support#3146] [Q&A]
When creating a standalone application, when creating .air for PC, by placing the file to be used in the bin folder, the file was included and read in the build.
However, in the case of the .apk for Android, even though it contained files when looking at the capacity, I could not read the file in the same way as for PC.
I am using a method such as File.applicationDirectory.nativepath to obtain the path and read from there, but in the case of Android, is it possible to read by changing the path to something else?
Also, do I have to think of another method?

スタンドアロン型のアプリを作成する際、PC用の.airを作成した際にはbinフォルダ内に使用するファイルを設置しておくことで、ビルドにもそのファイルが含まれ読み込むことができました。
でも、Android用の.apkの場合、容量を見るとファイルが含まれているにもかかわらず、PC用と同じ方法でファイルを読み込むことが出来ませんでした。
File.applicationDirectory.nativepathなどでパスを取得してそこから読み込む方式をとっていますが、Androidの場合パスを別のものにすれば読み込むことが出来るでしょうか？
また、別の方法を考えなければいけないでしょうか？

**Comment by ajwfrost** [comment]
Hi

It should work the same with Android, or at least: `File.applicationDirectory.resolvePath("myFile.txt")` should give you a path that you can use. The `nativePath` value should show you a temporary/cache location, because we have to pull the files out of the APK at runtime.

But: I think you are being impacted by a bug in AIR. We had found the same thing recently which was caused by the format of the APK file and some poor coding within the AIR runtime classes. We're about to release a new version which should fix this, so please check back early next week and try again with 50.2.4.5.

thanks

**Comment by HirokatsuIshikawa** [comment]
I was able to successfully retrieve the file using the specified method. thank you very much.
However, after changing from version 50.2.3.6 that I had used in the past to version 50.2.4.5, the contents of the variables being debugged that were displayed in remote debugging no longer appeared.
Will this be a bug? Or will the specifications change?
The development environment uses FlashDevelop.

指定された方法で無事にファイルを取得することができました。どうもありがとうございます。
ただ、過去に使っていたバージョン50.2.3.6から50.2.4.5に変更してから、リモートデバッグで表示されていたデバッグ中の変数の内容が出てこなくなってしまいました。
こちらはバグになるでしょうか？　それとも仕様変更になるでしょうか？
開発環境はFlashDevelopを使用しております。
